Orso.Arpa.Persistence/Migrations/20230622144827_AddNewsTitle.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Orso.Arpa.Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddNewsTitle : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
```

Comment:
Bitte noch in der Configuration die maxlength festlegen und die migration erneut ausführen (bzw. die alte rückgängig machen)